api/db/activity/statePersonnel.js

```diff
@@ -15,12 +15,13 @@ module.exports = {
         id: this.get('id'),
         title: this.get('title'),
         description: this.get('description'),
+        keyPersonnel: this.get('key_personnel'),
```

Comment:
The activity OpenAPI component will need to be updated to include this.

Reply:
👍